src/pyaro_readers/merging_reader.py

```diff
@@ -116,26 +116,61 @@ def __init__(
             )
 
     def _unfiltered_data(self, varname: str) -> Data:
+        raise MergingReaderException("This method should not be called")
+
+    def data(self, varname: str) -> Data:
+        """This method is deliberately overridden to prevent
+        double filtering of the data
+        """
```

Comment:
I would set these lines as comments, not as documentation. How `data` is implemented is not important for the user, only for the programmer.

Reply:
Thanks, I've changed this to line comments